### PR TITLE
fix: remove vendor-prefixed props/rules when standard is supported, c…

### PR DESCRIPTION
**Pull request by bsmth**
…lean up box decoration break prop

Follow-up from https://github.com/mdn/interactive-examples/pull/2812

### Description

Some of the interactive examples had vendor-prefixed props in cases that weren't necessary anymore. This PR follows up on the cleanup to touch the pages to refresh them. Additionally removing screenshots of examples and adding as live samples instead.

### Related issues and pull requests

- [x] https://github.com/mdn/interactive-examples/pull/2812

### CODE REVIEW

File: files/en-us/web/css/_colon_autofill/index.md [hunk 1]
```diff
@@ -31,19 +31,16 @@ The **`:autofill`** CSS [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) match
 
 ## Examples
 
-The following example demonstrates the use of the `:autofill` pseudo-class to change the border of a text field that has been autocompleted by the browser. For the best browser compatibility use both `:-webkit-autofill` and `:autofill`.
+The following example demonstrates the use of the `:autofill` pseudo-class to change the border of a text field that has been autocompleted by the browser.
+For browser compatibility, both `:-webkit-autofill` and `:autofill` are matched using a forgiving selector list with {{cssxref(":is()")}}.
```
Comment (estelle) [inline files/en-us/web/css/_colon_autofill/index.md:35]:
```suggestion
To ensure we don't creat an [invalid selector lis](/en-US/docs/Web/CSS/Selector_list#invalid_selector_list), both `:-webkit-autofill` and `:autofill` are matched using a forgiving selector list with {{cssxref(":is()")}}.
```

File: files/en-us/web/css/box-decoration-break/index.md [hunk 2]
```diff
@@ -57,70 +57,112 @@ The `box-decoration-break` property is specified as one of the keyword values li
...
-Note that stacking these pieces vertically will result in the non-fragmented rendering.
-
-Now, the same example but styled with `box-decoration-break: clone` results in:
+The following example shows how block elements with box decoration look when they contain line breaks in a [multi-column layout](/en-US/docs/Learn/CSS/CSS_layout/Multiple-column_Layout).
+Notice how the result of `box-decoration-break: slice` would be the equivalent of the first {{htmlelement("div")}} if stacked vertically.
+
+```css hidden
+body {
+  background-color: grey;
+  font-family: sans-serif;
```
Comment (estelle) [inline files/en-us/web/css/box-decoration-break/index.md:114]:
```suggestion
```
or change line 122 to `font-size: 20px`

**Review by estelle — APPROVED** [review]
two nits. 

Thanks! 🎉

**Comment by bsmth** [comment]
Super, thank you!

**Pull request merged**